### PR TITLE
Apply gray background for code-block:: console

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
# Purposes
- Easy to understand which is command and output.
- Easy to copy and paste commands.

スクリーンショット 2020-03-05 22 15 00

### CODE REVIEW

**Comment by toshihikoyanase** [comment]
Thank you for your PR.
It improves readability. IMO, it would be more helpful if it shows which is which.

For instance, the following part of the tutorial just arranges the code cells.

image

How about adding some short descriptions like `Out:` before cells?

Example:
image

**Comment by keisuke-umezawa** [comment]
@toshihikoyanase Thank you for the suggestion. I applied it.

File: docs/source/tutorial/first.rst [hunk 1]
```diff
@@ -72,44 +78,64 @@ Let us clarify the terminology in Optuna as follows:
 In Optuna, we use the study object to manage optimization. Method :func:`~optuna.study.create_study` returns a study object.
 A study object has useful properties for analyzing the optimization outcome.
 
+To get the best params:
```
Comment (toshihikoyanase) [inline docs/source/tutorial/first.rst]: At L52, you used `best parameter` and I suggest the following change:
```suggestion
To get the best parameter:
```

File: docs/source/tutorial/first.rst [hunk 2]
```diff
@@ -63,33 +78,80 @@ Let us clarify the terminology in Optuna as follows:
...
+
+.. code-block:: console
+
+    FrozenTrial(number=26, state=<TrialState.COMPLETE: 1>, params={'x': 1.9926578647650126}, user_attrs={}, system_attrs={'_number': 26}, value=5.390694980884334e-05, intermediate_values={}, datetime_start=datetime.datetime(2018, 5, 9, 10, 23, 0, 87060), datetime_complete=datetime.datetime(2018, 5, 9, 10, 23, 0, 91010), trial_id=26)
+
+To get all trials:
+
+.. code-block:: python
+
+    study.trials  # all trials
```
Comment (hvy) [inline docs/source/tutorial/first.rst]: A nit. It's now more or less covered by the accompanying text.
```suggestion
    study.trials
```

File: docs/source/tutorial/first.rst [hunk 3]
```diff
@@ -63,33 +78,80 @@ Let us clarify the terminology in Optuna as follows:
...
 
 By executing :func:`~optuna.study.Study.optimize` again, we can continue the optimization.
 
 .. code-block:: python
 
-    In [10]: study.optimize(objective, n_trials=100)
-    ...
+    study.optimize(objective, n_trials=100)
+
+.. code-block:: python
```
Comment (hvy) [inline docs/source/tutorial/first.rst]: How about adding some text before this block or merge this block with the previous one? For the former, something along "To get the updated number of trials.".

**Comment by keisuke-umezawa** [comment]
Thank you for the reviews. I applied them.

**Pull request merged**